combat-trainer.lic

```diff
@@ -1727,7 +1727,7 @@ class TrainerProcess
     @skill_map = { 'Hunt' => 'Perception', 'Pray' => 'Theurgy', 'PercMana' => 'Attunement',
                    'Perc' => 'Attunement', 'Astro' => 'Astrology', 'App' => 'Appraisal', 'PrayerMat' => 'Theurgy',
                    'App Quick' => 'Appraisal', 'App Careful' => 'Appraisal', 'Tactics' => 'Tactics', 'Analyze' => 'Tactics',
-                   'Scream' => 'Bardic Lore', 'Perc Health' => 'Empathy', 'Khri Prowess' => 'Debilitation',
+                   'Scream' => 'Bardic Lore', 'Perc Health' => 'Empathy', 'Khri Prowess' => 'Debilitation', 'Teach' => 'Scholarship',
```

Comment:
❓ Is the goal for the teacher to learn Scholarship or the teachee to learn the skill? I.e. should this be controlled by a timer or something other than skill experience like 'Favor Orb'?

Reply:
its just to trickle learn scholarship and the skill you pick for both parties, mostly useful for people who don't leave combat

Reply:
:+1: